dennis/tasks/release.py

```diff
@@ -95,8 +91,7 @@ def run(self):
             release = self.github_repo.create_git_tag_and_release(
```

Comment:
Since you are here already... would you mind fixing an idempotency issue that we spotted just today:

"master" should get merged back into "develop" **before** creating a tag. Creating a tag+release should be the very last step, declaring that the release is done and dusted in all respects.